pkg/apis/config/types.go

```diff
@@ -47,6 +49,26 @@ type ETCD struct {
 	Backup ETCDBackup
 }
 
+// CSI configuration for all csi components
+type CSI struct {
+	// CSIAttacher is the configuration for the external-attacher
+	CSIAttacher *CSIAttacher
+}
+
+// CSIAttacher configuration for the external csi attacher
+type CSIAttacher struct {
+	// Timeout Timeout of all calls to the container storage interface driver.
```

Comment:
Can you make the doc strings more fluid and mention the defaults (if any)? E.g. this one could be:
```suggestion
	// Timeout specifies a timeout of all calls to the container storage interface driver. Defaults to 3m.
```